streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopicManagerTest.java

```diff
@@ -108,7 +111,8 @@ public void shouldReturnCorrectPartitionCounts() {
             topic,
             Collections.singletonList(new TopicPartitionInfo(0, broker1, singleReplica, Collections.emptyList())),
             null);
-        assertEquals(Collections.singletonMap(topic, 1), internalTopicManager.getNumPartitions(Collections.singleton(topic)));
+        assertEquals(Collections.singletonMap(topic, 1),
+                internalTopicManager.getNumPartitions(Collections.singleton(topic), new HashSet<String>(), 1));
```

Comment:
Could use Collections.emptySet() if reduced to Set

Reply:
Updated. Thanks.